CMakeLists.txt

```diff
@@ -1,75 +1,85 @@
-cmake_minimum_required(VERSION 3.1.3)
+cmake_minimum_required(VERSION 3.10.2)
 project(moveit2_tutorials)
 
-set(CMAKE_CXX_STANDARD 14)
+if(NOT "${CMAKE_CXX_STANDARD}")
+  set(CMAKE_CXX_STANDARD 14)
+endif()
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
 set(CMAKE_CXX_EXTENSIONS OFF)
 
-find_package(catkin REQUIRED
-  COMPONENTS
-    interactive_markers
-    moveit_core
-    moveit_ros_planning
-    moveit_ros_planning_interface
-    moveit_ros_perception
-    rviz_visual_tools
-    moveit_visual_tools
-    pluginlib
-    geometric_shapes
-    pcl_ros
-    pcl_conversions
-    rosbag
-    tf2_ros
-    tf2_eigen
-    tf2_geometry_msgs
-)
-
 find_package(Eigen3 REQUIRED)
 find_package(Boost REQUIRED system filesystem date_time thread)
+find_package(ament_cmake REQUIRED)
+find_package(moveit_core REQUIRED)
+find_package(moveit_ros_planning REQUIRED)
+find_package(moveit_ros_planning_interface REQUIRED)
+find_package(moveit_ros_perception REQUIRED)
+find_package(interactive_markers REQUIRED)
+#find_package(rviz_visual_tools REQUIRED)
+#find_package(moveit_visual_tools REQUIRED)
+find_package(geometric_shapes REQUIRED)
+#find_package(pcl_ros REQUIRED)
+#find_package(pcl_conversions REQUIRED)
+#find_package(rosbag REQUIRED)
+find_package(rclcpp REQUIRED)
+find_package(rclcpp_action REQUIRED)
+find_package(pluginlib REQUIRED)
+find_package(std_srvs REQUIRED)
+find_package(tf2 REQUIRED)
+find_package(tf2_ros REQUIRED)
+find_package(tf2_eigen REQUIRED)
+find_package(tf2_geometry_msgs REQUIRED)
 
 set(THIS_PACKAGE_INCLUDE_DIRS
+  ament_cmake
```

Comment:
Sorry I am not the best at cmake and was using the move_group.cmake to help me port this. Any help here would be appreciated.

probably need to remove the following from this list:
`std_srvs`
`moveit_ros_occupancy_map_monitor`
`moveit_ros_planning`
`pluginlib`